python/lsst/ts/wep/task/calcZernikesUnpairedTask.py

```diff
@@ -109,26 +148,35 @@ def run(self, donutStamps: DonutStamps) -> pipeBase.Struct:
             donutQualityTable = QTable([])
 
         # Assign stamps to either intra or extra
-        if selectedDonuts[0].wep_im.defocalType == DefocalType.Extra:
+
+        defocalType = donutStamps.metadata["DFC_TYPE"]
+        if defocalType == "extra":
             self.stampsExtra = selectedDonuts
-            extraStamps = selectedDonuts
-            intraStamps = DonutStamps([])
+            self.stampsIntra = None
```

Comment:
why are you defining this as None at all? It doesn't seem to be used

Reply:
It's used when we run `createUnpairedZernikeTable` below.